book/website/collaboration/ppie-events.md

```diff
@@ -0,0 +1,60 @@
+- {ref}`initial planning and scoping guidance<cl-ppie-events-planning>`
+- {ref}`guidance on how to evaluate and layout your venue<cl-ppie-events-location>`
+- {ref}`considerations and requirement for developing a communications strategy and how to support your speakers<cl-ppie-events-comms>`
+- {ref}`suggestions for how to create an inclusive agenda<cl-ppie-schedule-planning>`
+- {ref}`considerations for how to make technology work for your events<cl-ppie-events-technology>`
+- {ref}`suggestions for how to plan social or networking sessions<cl-ppie-events-social-and-networking-guide>`
+- and provides a checklist that can be tailored to your specific event
+
+
+(cl-ppie-events-background)=
+## Motivation and Background
+
+  * This chapter is meant to serve as a resource to support and guide researchers, organisers, and community members with the tools to identify the opportunities and mitigate barriers in the organisation of events that involve and include members of the public, and in particular people with lived experience.
+
+We refer to "people with lived experience" throughout this chapter as a way to recognize and acknowledge individuals who have direct personal knowledge, or firsthand experience of a particular situation that is relevant to the topic of a research-related event. 
+For example, this could relate to a particular medical condition, disability, or other social condition. 
+It is a term often used to emphasise the importance of including the perspectives and insights of those who have had direct experience. {cite:ps}nihr2021differentexperiences,camden2022preferredlanguage
+
+This chapter is focused in particular on how to make research events more inclusive and accessible to members of the public and people with lived experience who do not have experience or a background as a researcher, as they can face particular barriers to participation. 
+
+Many research projects are increasingly including members of the public in the research process. 
+Within the realm of biomedical and care-related research, this is often referred to as "Public and Patient Involvement and Engagement" or "PPIE" which has a goal to actively involve people with the lived experience of a specific disability, situation, health condition, or experience in the research process. {cite:ps}HDRUK2023ppie 
+PPIE is done both to ensure that research is aligned with the interests of people affected by it, and to improve the quality of research by including lived knowledge and perspectives. 
+As part of this involvement and engagement, researchers and research staff will need to plan events that are inclusive and accessible for people with lived experience, and delivered in a way that avoids presenting barriers to people with lived experience taking part equitably.
+
+"Lived experience" can be a range of things such as a disability, long-term health condition, bereavement, having worked in a specific profession, or the lived experience of caring responsibilities. 
```

Comment:
```suggestion
"Lived experience" as it is used within the research context can be a range of things such as a disability, long-term health condition, bereavement, having worked in a specific profession, or the lived experience of caring responsibilities. 
```
Adding a gentle suggestion here for some qualifying language. Should this be added to the glossary?

Reply:
Good suggestion! I think it would be a good idea to enter even a basic definition in. 

Do you think this sentence should also reference how the phrase lived experience is used/means within a policy, social, and health care settings too? Or do we need to narrow it down to just be relevant to the research experience?